sql/core/src/test/resources/sql-tests/inputs/cast.sql

```diff
@@ -70,6 +70,11 @@ select cast(' 1' as bigint);
 select cast(' 1' as float);
 select cast(' 1 ' as DOUBLE);
 select cast('1.0 ' as DEC);
+select cast('1中文' as tinyint);
```

Comment:
This is purely to educate me, but those characters are considered whitespace?

Reply:
This kind of character needs multiple bytes, so `getByte(s) <= ' '` may not work.